modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
+
+The Salesforce platform uses the System Administrator profile which contains permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
```

Comment:
```suggestion
The Salesforce platform uses the System Administrator profile, which includes permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
```

Reply:
I feel like "AEH_Anypoint" should be in a different font - probably because the underbar makes it kind of codey.

Reply:
Which font do you suggest?

Reply:
Usually, we either put it in mono font or remove the underbar, but if that's how Salesforce handles permission sets, then it's fine.